components/support/support.jsx

```diff
@@ -1,56 +1,49 @@
 import React from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
Sure... I was leaving it in for now as kind of a TODO. I'd like to add prop type checking to some of our components at some point but I'm fine with curbing this for now.